modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Filters.tsx

```diff
@@ -167,10 +205,7 @@ function AddFDSFilterModalContent({
 		let displayType: string = '';
 		let url: string = '';
 
-		if (
-			selectedField.format === 'date' ||
-			selectedField.format === 'date-time'
-		) {
+		if (selectedField.format === fieldFormats.DATE_TIME) {
```

Comment:
is `'date'` still a possibility?

Reply:
yep, we'd need both. 

Unfortunately, there are other associated issues when using date time (odata parsing and lack of timepickers in the FDS) requiring specific, separate tasks to deal with them. What works ootb at this moment is date format, so we need it

Reply:
ok, I'll restore 'date' type